### PR TITLE
Cluster list handling code

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Cluster list handling code

## Type of change

- New feature (non-breaking change which adds functionality)
- Documentation update

## Testing steps

N/A

### CODE REVIEW

**Review by Bee-lee — APPROVED** [review]
lgtm

File: http/router_utils.go [hunk 1]
```diff
@@ -247,3 +249,41 @@ func handleClusterNameError(writer http.ResponseWriter, err error) {
 func SplitRequestParamArray(arrayParam string) []string {
 	return strings.Split(arrayParam, ",")
 }
+
+// ReadClusterListFromPath retrieves list of clusters from request's path
+// if it's not possible, it writes http error to the writer and returns false
+func ReadClusterListFromPath(writer http.ResponseWriter, request *http.Request) ([]string, bool) {
+	rawClusterList, err := GetRouterParam(request, "cluster_list")
```
Comment (joselsegura) [inline http/router_utils.go:256]: According to https://github.com/RedHatInsights/insights-results-smart-proxy/blob/ebf45c0dba7e31516b1fd7d3696fd28d91e6562c/openapi.json#L366, it should be `clusterList`

**Review by bond95 — APPROVED** [review]

File: http/router_utils.go (hunk 1 above)
Comment (joselsegura) [reply http/router_utils.go:256]: It's a nitpick, probably it is better to change the openapi.json spec rather than this

Comment (tisnik) [reply http/router_utils.go:256]: yup, will do

**Pull request merged**